requirements.txt

```diff
@@ -2,3 +2,4 @@ pulpcore>=3.17.0.dev
 pkginfo
 packaging
 bandersnatch>=5.0.0
+pypi-simple
```

Comment:
Does this need to go in `requirements.txt` if it's already in `functests_requirements.txt`?

Reply:
Yes. The package doesn't pull from `functests_requirements.txt` when building its dependencies. A method from this package is also used in the pull-through methods.